src/backend.md

```diff
@@ -630,6 +630,55 @@ For serverless projects - method 3 is preferred, but not always. When you have a
       - updateBankAccount.controller.test.ts
 ```
 
+1. One test should include one action as a call of controller or job.
```

Comment:
🏞️ It's a part of [the AAA testing pattern](https://www.oreilly.com/library/view/software-architects-handbook/9781788624060/a6b2159e-022c-459c-9a3d-5a1b34886772.xhtml).
A test with one act is more stable and simple than with many acts.